### ISSUE TITLE
gulp build error

### ISSUE THREAD

**Issue by Awakening-j** [randylien/generator-react-gulp-browserify#52]
```js
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: spawn sass ENOENT
    at exports._errnoException (util.js:746:11)
    at Process.ChildProcess._handle.onexit (child_process.js:1053:32)
    at child_process.js:1144:20
    at process._tickCallback (node.js:355:11)
```  

  
gulp build error, is this a project bug?

**Comment by randylien** [comment]
Hi @Awakening-j, one possible reason because I forgot to add babelify into package.json.
You can run `npm install babelify` in your project folder to fix it.

**Comment by Awakening-j** [comment]
@randylien Thanks

**Comment by Awakening-j** [comment]
@randylien still has issue   
```js
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: spawn sass ENOENT
    at exports._errnoException (util.js:746:11)
    at Process.ChildProcess._handle.onexit (child_process.js:1053:32)
    at child_process.js:1144:20
    at process._tickCallback (node.js:355:11)
```  

and run gulp test shows:  
```
Using Jest CLI v0.4.15
 FAIL  app/scripts/ui/__tests__/Timer-test.js
Error: /Users/working/Documents/projects/demo/react-gulp/app/scripts/ui/__tests__/Timer-test.js: Cannot find module '../Timer' from '/Users/working/Documents/projects/demo/react-gulp/app/scripts/ui/__tests__/Timer-test.js'
1 test failed, 0 tests passed (1 total)
```

**Comment by randylien** [comment]
@Awakening-j sass issue should be fixed 0.0.29. Please try it.

**Comment by Awakening-j** [comment]
@randylien ok, thanks:)
